manifests/example-template.yml

```diff
@@ -33,9 +33,7 @@ spec:
             type: network
           video:
           - type: qxl
-        memory:
-          unit: MB
-          value: 64
+        memory: 64M
```

Comment:
Note that this is 64\*10^6 bytes. You may want to use 64MiB which is 64\*2^30 bytes.